### ISSUE TITLE
Warnings on upgrading to Terraform v0.11.1

### ISSUE THREAD

**Issue by horsey** [terraform-aws-modules/terraform-aws-rds#31]
```Warning: output "this_db_parameter_group_id": must use splat syntax to access aws_db_parameter_group.this attribute "id", because it has "count" set; use aws_db_parameter_group.this.*.id to obtain a list of the attributes across all instances



Warning: output "this_db_parameter_group_arn": must use splat syntax to access aws_db_parameter_group.this attribute "arn", because it has "count" set; use aws_db_parameter_group.this.*.arn to obtain a list of the attributes across all instances

**Comment by antonbabenko** [comment]
Thanks for reporting! This is an important issue to fix in this and several other AWS modules also.

**Comment by horsey** [comment]
Probably not related to this issue, but is there a way to pin my infrastructure to a version of the module?  I happen to delete my .terraform directory and discovered that I needed to upgrade my Terraform version because it was not compatible with the latest version of the module.

**Comment by antonbabenko** [comment]
Yes, you can specify [version](https://www.terraform.io/docs/configuration/modules.html#version) like this:
```hcl
module "db" {
  source = "terraform-aws-modules/rds/aws"
  version = "v1.4.0"

  # the rest of arguments...
}
```

Here you can see all releases - https://github.com/terraform-aws-modules/terraform-aws-rds/releases

**Comment by horsey** [comment]
Thank you!

**Comment by horsey** [comment]
@antonbabenko Any plans to make a release with the fix? 

Thanks in advance.

**Comment by antonbabenko** [comment]
This is now fixed in [v1.5.0](https://github.com/terraform-aws-modules/terraform-aws-rds/releases/tag/v1.5.0)
